### PR TITLE
Add a .nvmrc to the VZ apps

### PR DESCRIPTION
**Pull request by roseeichelmann**
## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/11023

This PR adds support for [Node Version Manager](https://github.com/nvm-sh/nvm) to the VZV. It looks like @frankhereford already added a .nvmrc file to VZE so thank you 🙏

## Testing
**URL to test:** 
Locally & [netlify](https://deploy-preview-1199--atd-vzv-staging.netlify.app/)

**Steps to test:**
1. To test locally follow steps to set up nvm in the [README](https://github.com/cityofaustin/atd-vz-data/blob/1cde80fe24aff9f3ae2f6923d85f8e89dd75164f/atd-vzv/README.md) of each directory
2. When you add a .nvmrc file to your repository Netlify automatically builds using the version you specified there (pretty cool!). If you check the [build logs](https://app.netlify.com/sites/atd-vzv-staging/deploys/641b46a21022e0000824f424#L54) you can see how it checks .nvmrc and uses v16.

If you want `nvm use` to auto-run whenever you switch directories, I found these [instructions](https://github.com/nvm-sh/nvm#deeper-shell-integration) to set that up

---
#### Ship list
- [ ] Code reviewed
- [x] Product manager approved

### CODE REVIEW

File: atd-vze/README.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ For resources and updates, see the [Vision Zero Crash Data System](https://githu
 
 ## Getting started
 
+We use [Node Version Manager](https://github.com/nvm-sh/nvm) (nvm) to keep our `node` versions in sync with our environments. With `nvm` installed, run `nvm use` from this directory to activate the current `node` and `npm` version required for this project. If you don't want to use `nvm`, refer to the `.nvmrc` file for the `node` version you should install.
+
```
Comment (roseeichelmann) [inline atd-vze/README.md:12]: stole this word for word from @johnclary in this [PR](https://github.com/cityofaustin/atd-moped/pull/688) 🤪

**Comment by frankhereford** [comment]
@roseeichelmann wrote:

> [end-of-life](https://github.com/nodejs/release#release-schedule) for node v16 is in september 2023

Good catch!! Thank you @roseeichelmann!

The VZE doesn't currently build with this version of node because they pulled out some core libs from it, IIRC. It would take a little bit of work to get those added back as dependencies, maybe a `2` sized task?

**Comment by roseeichelmann** [comment]
@frankhereford ah thank you for that extra info, yeah that sounds like it could be a good backlog issue - to pin VZE and VZV to v18 and add back in dependencies. not super high priority but would be good to address it before the EOL!

**Review by frankhereford — APPROVED** [review]
Thanks for helping standardize the environment we expect the app to run in. I think that is very helpful!

> If you want nvm use to auto-run whenever you switch directories, I found these [instructions](https://github.com/nvm-sh/nvm#deeper-shell-integration) to set that up

This is such a pro tip, thank you! ✨

File: atd-vze/README.md (hunk 1 above)
Comment (mddilley) [reply atd-vze/README.md:12]: I remember that I mentioned [fnm](https://github.com/Schniz/fnm) in that PR. Ended up switching over, and it works really well. Almost instant Node version change but I really don't think we are sweating fractions of seconds so... 🙃

**Review by mddilley — APPROVED** [review]
So nice to have this set in here - thank you for this and the reminder about Node 18! 🚀

**Review by patrickm02L — APPROVED** [review]
🚢 Ship it!

**Review by chiaberry — APPROVED** [review]

**Pull request merged**